test-app/src/generated-types.ts

```diff
@@ -0,0 +1,182 @@
+import { GraphQLResolveInfo } from 'graphql';
```

Comment:
What's best practice here? Check in the file or not?

Reply:
So I just saw this [Paul Swail post](https://serverlessfirst.com/emails/graphql-codegen-for-appsync-projects/) that advises against checking in the generated types. In my projects I've personally checked them in because I like to see type changes in pull requests. I think we could choose not to check them but we would need to make sure that the codegen script is run before build and deploy, maybe even on yarn install? So that the user does not need to run yarn codegen manually after cloning the repo.

Reply:
I understand what you mean - like a snapshot test. Let's go with it - it'll be big on the first commit but changes should be irregular.

Reply:
Also, it's good for our sample app to show them for people browsing.